lte/gateway/c/session_manager/StoredState.h

```diff
@@ -218,6 +218,7 @@ struct StoredSessionState {
   uint32_t local_teid;
   uint64_t pdp_start_time;
   uint64_t pdp_end_time;
+  CreateSessionResponse create_session_response;
```

Comment:
this will store the response received by LTE core.
Its content will be deleted once it is used at the end of `UpdateTunelIds`

Reply:
Might want to put that into comments